.circleci/config.yml

```diff
@@ -1,59 +0,0 @@
-version: 2.1
-
-jobs:
-  build:
-    working_directory: ~/slovensko-digital/navody.slovensko.digital
-
-    docker:
-      - image: circleci/ruby:2.7.2-node-browsers
-        environment:
-          CC_TEST_REPORTER_ID: 17d259e00bbff5b020b7f1a6abb6736e0f65b9a517afdcbaeaa79e919730835f
```

Comment:
toto som pridal do "secrets" a teda asi to netreba uz nastavovat.

Reply:
aby bolo jasne je to ako `CC_TEST_REPORTER_ID` secret